apps/dashboard_v2/api.py

```
@@ -42,8 +34,8 @@ async def get_dashboard_overview(
             logger.info(f"Dashboard overview retrieved for tenant {tenant_id}")
             return overview
     except Exception as e:
-        logger.error(f"Error getting dashboard overview: {e}")
-        raise HTTPException(status_code=500, detail="Failed to retrieve dashboard overview")
+        logger.exception(f"Error getting dashboard overview: {e}")
```

Ruff (0.13.3)

Redundant exception object included in `logging.exception` call

(TRY401)